tests/opentitan/opentitan-210214/Makefile.in

```diff
@@ -0,0 +1,3 @@
+OPENTITAN_VERSION = opentitan-210214
```

Comment:
Is it possible to move ``OPENTITAN_VERSION`` variable to verilator CI build matrix and move this file (with only):
```
include $(root_dir)/../uhdm-tests/opentitan/Makefile.in
```
to ``tests/opentitan``?

Reply:
I was considering that, but then we need to specify this variable not only in the CI but also for manual calls. Note that the test name (derived from location) already specifies the Opentitan version required:
```
TEST=tests/opentitan/opentitan-210214 make uhdm/verilator/opentitan
```
With additional variable we end up duplicating the version information:
```
OPENTITAN_VERSION=opentitan-210214 TEST=tests/opentitan/opentitan-210214 make uhdm/verilator/opentitan
```
I've opted for avoiding duplication, unless there is a scenario that requires this?

Reply:
I think in the second approach the call would be:
```
OPENTITAN_VERSION=opentitan-210214 TEST=tests/opentitan make uhdm/verilator/opentitan
```
so there won't be any duplication of the information.

It is true, that we would need to specify it in manual calls and it changes typical use-case, but I think it is better to add this variable anyway.

I'm asking about this, because, if we will create this folders for version of the opentitan (without Makefile inside ``opentitan`` folder) it would require to add current version of opentitan that is used in yosys (it probably isn't ``opentitan-210214`` and ``opentitan-current``).

Reply:
That's true for the generic target, but unfortunately the module tests are version-specific and need to stay in their respective folders.

I've added the makefile you requested for generic `opentitan` target to the folder, with clarification that exact version will be tool-specific. This way we both avoid the duplication and are ready to add specific version(s) for yosys when needed.